openreview/arr/arr.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, client, venue_id, support_user, venue=None):
         self.use_publication_chairs = False
         self.source_submissions_query_mapping = {}
         self.sac_paper_assignments = False
+        self.reviewer_assignment_quota = None
```

Comment:
can we call this `submission_assignment_max_reviewers` so it matches with the name convention of the other quotas?

I also think that `reviewer_assignment_quota` can be interpreted as the max number of papers to assign to a reviewer.